modules/extensions-support/src/main/java/org/mule/module/extension/internal/manager/DefaultExtensionManager.java

```diff
@@ -50,6 +58,7 @@
     private ObjectNameHelper objectNameHelper;
     private ExtensionDiscoverer extensionDiscoverer = new DefaultExtensionDiscoverer(new DefaultExtensionFactory(serviceRegistry), serviceRegistry);
     private ImplicitConfigurationFactory implicitConfigurationFactory;
+    private ConfigurationExpirationMonitor configurationExpirationMonitor;
```

Comment:
we discussed this with LG and found no use case which justifies having N amount of expiry threads running. Let's discuss this offline if you do have a case